osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -44,6 +44,11 @@ namespace osu.Game.Rulesets.Edit
     public abstract partial class HitObjectComposer<TObject> : HitObjectComposer, IPlacementHandler
         where TObject : HitObject
     {
+        /// <summary>
+        /// Whether the playfield should be centered vertically. Should be disabled for playfields which span the full horizontal width.
+        /// </summary>
+        protected virtual bool ApplyVerticalCentering => true;
```

Comment:
Okay, first of all, this applies *horizontal* centering, and not vertical.

Reply:
Hmm. That depends how your brain works. But I think you're correct that that is the more accepted understanding.

Reply:
Not sure why this rename wasn't applied, but given that the above seems to be an agreement, I'll apply it myself to speed things up because god knows I'm review-backlogged to my ears already.